bundles/AdminBundle/Controller/Admin/TagsController.php

```diff
@@ -95,6 +95,43 @@ public function updateAction(Request $request)
         }
     }
 
+    /**
+     * @param Request $request
+     * @Route("/update-translation-for-tag")
+     */
+    public function updateTranslationForTagAction(Request $request){
+        $success = false;
+
+        if($tag = Tag::getById($request->get('id'))){
+            $translations = $tag->getTranslations();
+            $translations[$request->get('language')] = $request->get('translation');
+
+            $tag->setTranslations(serialize($translations));
```

Comment:
Maybe we could even add an `addTranslation(string $language, string $translation)` method in the model?

Then this would only be:
```suggestion
            $tag->addTranslation($request->get('language'), $request->get('translation'));
```